earthly/python/Earthfile

```diff
@@ -22,23 +28,40 @@ python-base:
 
     # Stop python code running in the container saving pointless .pyc files.
     ENV PYTHONDONTWRITEBYTECODE=1
-
+    
     # Install Poetry to manage python.
     RUN curl -sSL https://install.python-poetry.org | python3 -
     # Prevent warning when running on machines with more than 6 cores.
     #   "Connection pool is full, discarding connection: pypi.org. Connection pool size: 10"
     RUN poetry config installer.max-workers 10
+    
+    # Install flake8 globally using Poetry
+    RUN pip3 install flake8
```

Comment:
We don't want to use flake8 as the linter.  We should be using `ruff` as its much much faster, includes flake8 checks and a ton of others all in the same tool.
See: https://github.com/astral-sh/ruff

Sorry, it should have been stated in the ticket.

Reply:
Changed to ruff commit c90df2f16b81271f0563faeccfa1d25161ca46af